pkg/storage/unified/resource/search.go

```diff
@@ -890,7 +882,7 @@ func (s *searchSupport) build(ctx context.Context, nsr NamespacedResource, size
 		attribute.Int64("size", size),
 	)
 
-	logger := s.logWithTraceID(ctx).With("namespace", nsr.Namespace, "group", nsr.Group, "resource", nsr.Resource)
+	logger := s.log.New("namespace", nsr.Namespace, "group", nsr.Group, "resource", nsr.Resource)
```

Comment:
I removed trace info from here, as this `logger` gets used during index updates too, and those are unrelated to the original request that triggered index build.